packages/big-design/src/components/StatefulTable/spec.tsx

```diff
@@ -43,21 +43,20 @@ const getSimpleTable = (props: Partial<StatefulTableProps<TestItem>> = {}) => (
   />
 );
 
-test('renders a non-paginated table by default', () => {
-  const { container } = render(getSimpleTable());
+test('renders a non-paginated table by default', async () => {
+  const { findAllByRole } = render(getSimpleTable());
 
-  const rows = container.querySelectorAll('tbody > tr');
+  const rows = await findAllByRole('row');
```

Comment:
This now accounts for the `tr` in the `thead`. When I tried to filter out the `thead > tr` I ran into performance issues. Just bumped the number of rows by 1.